js/classes.js

```diff
@@ -101,6 +101,12 @@ class Task {
         }
         this.xp = this.getMaxXp() + excess;
     }
+
+    updateMaxLevelAndReset() {
+        if (this.level > this.maxLevel) this.maxLevel = this.level;

```

Comment:
```suggestion
        if (this.level > this.maxLevel) {
                this.maxLevel = this.level;
        }
```
Bitte alles was über ein `return;` hinausgeht ordentlich klammern